recipe/meta.yaml

```diff
@@ -32,30 +32,30 @@ requirements:
     - dask-jobqueue
     - distributed
     - esgf-pyclient >=0.3.1
-    - esmpy !=8.1.0
+    - esmpy !=8.1.0,<8.6.0  # https://github.com/SciTools-incubator/iris-esmf-regrid/pull/342#issuecomment-2092921514
     - filelock
     - fiona
     - fire
     - geopy
     - humanfriendly
-    - importlib_metadata
-    - iris >=3.6.1
+    - importlib_metadata  # required for Python < 3.10
+    - iris >3.8.0
     - iris-esmf-regrid >=0.7.0
     - isodate
     - jinja2
     - libnetcdf !=4.9.1  # issues lots of HDF5 warnings
     - nc-time-axis
     - nested-lookup
     - netcdf4
-    - numpy !=1.24.3  # severe bug in masked arrays
+    - numpy !=1.24.3,<2.0.0  # severe bug in masked arrays, avoid pulling 2.0.0
     - packaging
-    - pandas
+    - pandas !=2.2.0,!=2.2.1,!=2.2.2  # github.com/ESMValGroup/ESMValCore/pull/2305 and #2349
```

Comment:
@bouweandela note that this, together with the numpy pin will have to go too, in the near future - hopefully we'll allow numpy2 support before the next release (ahem, iris, ahem)